Makefile

```diff
@@ -46,7 +46,7 @@ lint:
 
 build:
 	@echo "+ $@"
-	@go build -v ${GO_LDFLAGS} ./...
+	@go build -v  -gcflags '-N -l' ${GO_LDFLAGS} ./...
```

Comment:
Want to make that controllable with an environment variable? What is the cost of leaving it on by default? (love and miss gdb).